app/assets/stylesheets/common/base/discourse.scss

```diff
@@ -203,8 +203,7 @@ input {
   &[type="url"],
   &[type="search"],
   &[type="tel"],
-  &[type="color"],
-  &.input-setting-integer {
```

Comment:
This selector was intentionally added a few days ago. We will want to keep it.

Reply:
It was only added to partially fix a recent regression. This PR fixes it completely so this is no longer needed. I'll extract it to a separate PR for clarity 🙂